docs/source/tutorials/debugging.ipynb

```diff
@@ -40,23 +52,25 @@
     "with rp.Session() as session:\n",
     "\n",
     "    pmgr  = rp.PilotManager(session=session)\n",
-    "    pilot = pmgr.submit_pilots(rp.PilotDescription({'resource': 'local.localhost', \n",
-    "                                                    'cores'   : 4,\n",
-    "                                                    'runtime' : 10}))\n",
-    "    \n",
     "    tmgr  = rp.TaskManager(session=session)\n",
+    "    \n",
+    "    pilot = pmgr.submit_pilots(\n",
+    "        rp.PilotDescription({'resource'     : 'local.localhost', \n",
+    "                             'cores'        : 4,\n",
+    "                             'runtime'      : 10,\n",
+    "                             'exit_on_error': False}))\n",
     "    tmgr.add_pilots(pilot)\n",
+    "\n",
     "    td_1  = rp.TaskDescription({'executable': 'date'})\n",
```

Comment:
we say above: 
```
We submit one task which suceeds (`task.000000`) and one with an invalid executable which fails (`task.000001`).
```

But aren't both tasks going to fail: 
```
    td_1  = rp.TaskDescription({'executable': 'date'})
    td_2  = rp.TaskDescription({'executable': 'data'})  # <- unknown executable
```

due to unknow executable?

Reply:
`date` will return a date, but there is no such executable as `data`, 
and these tasks do not depend on each other, so the failure of one task will not affect the other one